water-connection/ws-calculator/src/main/resources/db/migrate.sh

```
@@ -1,20 +1 @@
-do
-    echo "the schema name : ${baseurl}${schemasetter}${schemaname}"
-    flyway -url="${baseurl}${schemasetter}${schemaname}" \
-           -table="$SCHEMA_TABLE" \
-           -user="$FLYWAY_USER" \
-           -password="$FLYWAY_PASSWORD" \
-           -locations="$FLYWAY_LOCATIONS" \
-           -baselineOnMigrate=true \
-           -outOfOrder=true \
-           migrate
-done
+flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true -outOfOrder=true migrate
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)